### DISCUSSION TITLE
P1- DSMR settings

### DISCUSSION THREAD

**Discussion by yann1420** [arendst/Tasmota#12543] [Q&A] [answered]
Hello team,
I built  a Wemos mini according this page [](https://tasmota.github.io/docs/P1-Smart-Meter/ ) and it starts spitting information. Then I applied the commands

- SerialDelimiter 10
- SerialSend 1
and 
- SerialBuffer 520 
Yes, I know it needs a rule for after reboot, but when typed in the console I get the following message
```
08:53:48 CMD: SerialBuffer 520 
08:53:48 MQT: stat/tasmota82/RESULT = {"Command":"Unknown"}

```
The Wemos runs tasmota 8.3.1

What can be wrong?
Thanks in advance.

**Comment by sfromis** [comment, marked as answer]
Better upgrade to current Tasmota if you expect the documentation to apply. The SerialBuffer command was only introduced in april.
